src/index.html

```diff
@@ -1,12 +1,660 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;500;600&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header header--container">
+      <a href="#logo" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
```

Comment:
```suggestion
      <a href="#logo" class="header__logo">
        <img src="./images/logo.png" alt="logo">
      </a>
      <nav class="header__nav nav">
        <ul class="nav__list">
```

Your BEM-blocks are wrong. If you have nesting elements/blocks you need to add proper classnames.
[Typical BEM mistakes](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes)

Reply:
Тут теж суперечливо на мій погляд. Бо якщо ми напишемо header__logo чи  header__nav, то ми не зможемо використати їх в футері не змінивши назви цих класів

Reply:
@KorniychukLesya Yes, this part is totally how your decide it will work. In case of reuse of `logo` you need to create a new scss file in `blocks` called `logo.scss` and move style there cause it exists as separate BEM-block.

Reply:
Може тут ментор нас розсудить. Бо і робити окремий файл - занадто і додати до всіх елементів header__ -теж ніби не зовсім вірно. Може тут якусь потрібно додаткову обгортку зробити?

Reply:
@KorniychukLesya You can leave it as is, no problem 👌